nft_ingester/src/processors/transaction_based/bubblegum_updates_processor.rs

```diff
@@ -274,134 +281,126 @@ impl BubblegumTxProcessor {
 
         let instruction: Result<InstructionResult, IngesterError> = match ix_type {
             InstructionName::Transfer
+            | InstructionName::TransferV2
             | InstructionName::CancelRedeem
-            | InstructionName::Delegate => {
+            | InstructionName::Delegate
+            | InstructionName::DelegateV2
+            | InstructionName::DelegateAndFreezeV2
+            | InstructionName::FreezeV2
+            | InstructionName::SetNonTransferableV2
+            | InstructionName::ThawV2
+            | InstructionName::ThawAndRevokeV2 => {
                 Self::get_update_owner_update(parsing_result, bundle).map(From::from).map(Ok)?
```

Comment:
Given the nature of all these operations not being exclusively ownership/delegate changes, I'd suggest we rename the method to smth more relevant. May be get_cnft_properties_update or smth like that.